libs/service/api_service.cpp

```diff
@@ -547,6 +763,39 @@ ebpf_authorize_native_module(HANDLE native_image_handle) noexcept
     EBPF_RETURN_RESULT(result);
 }
 
+/**
+ * @brief Initialize the test signing state.
+ *
+ * @retval EBPF_SUCCESS on success, or an error code on failure.
+ * @retval EBPF_INVALID_ARGUMENT if the operation ID in the reply is not as expected.
+ * @retval EBPF_NO_MEMORY if memory allocation fails.
```

Comment:
retval generates a 2-column table (value & description).
```suggestion
 * @retval EBPF_SUCCESS The operation was successful.
 * @retval EBPF_INVALID_ARGUMENT The operation ID in the reply was not as expected.
 * @retval EBPF_NO_MEMORY Memory allocation failed.
```